component/Page/Profile/index.tsx

```diff
@@ -214,7 +215,7 @@ export default function ProfilePage(props: PageProps) {
                                     }}
                                 />
                             </div>
-                            <button className="rounded-full flex-shrink-0 bg-cyan h-8 w-8 flex items-center justify-center self-center" onClick={() => {
+                            <button className="rounded-full flex-shrink-0 bg-cyan h-8 w-8 flex items-center justify-center self-start" onClick={() => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `self-center` to `self-start` in the button class list could affect the positioning of the button. Make sure this change doesn't disrupt the layout or functionality of the button.
```diff
- <button className="rounded-full flex-shrink-0 bg-cyan h-8 w-8 flex items-center justify-center self-start"
+ <button className="rounded-full flex-shrink-0 bg-cyan h-8 w-8 flex items-center justify-center self-center"
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->